p2p/transport_test.go

```diff
@@ -1,696 +0,0 @@
-package p2p
```

Comment:
did this get moved somewhere?

Reply:
Mentioned this in the description:

>There are no tests of the new MConnTransport, as this code is likely to evolve as we proceed with the P2P refactor, but tests should be added before a final release. The E2E tests are sufficient for basic validation in the meanwhile.

E.g. all of the connection filtering stuff should be moved into the router, the message send/receive semantics will change slightly, and the interface will be reduced, so there's no point writing tests for this now.